utils/evaluate_utils.py

```diff
@@ -99,6 +100,64 @@ def class_PR_curves(
     return fig, PR_data
 
 
+def get_SCM_model_data(
```

Comment:
Just wanted to call out: nice design choice on decoupling this work from `class_PR_curves_SCM`!

Reply:
Thanks